### PR TITLE
Fixes neonlabsorg/neon-compatibility#107

### PR DESCRIPTION
**Pull request by gigimon**

### CODE REVIEW

**Comment by gigimon** [comment]
Allure Report 2021-11-15 11-11-04

File: .github/actions/allure-environment-data/action.yml [hunk 1]
```diff
@@ -6,63 +6,34 @@ runs:
...
+        echo "OpenZeppelin.Author=$TEST_AUTHOR" >> $ENV_PROP_FILE
         
         # Solana cluster version
         SOLANA_VERSION=$(curl $SOLANA_URL -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "method":"getVersion"}')
         SOLANA_VERSION=$(echo $SOLANA_VERSION | sed 's/\"/\\\"/g')
         echo "Solana.Version=$SOLANA_VERSION" >> $ENV_PROP_FILE
-        echo "{ \"name\": \"Solana.Version\", \"values\": [ \"$SOLANA_VERSION\" ] }" >> $ENV_FILE
-        echo "{ \"name\": \"Solana.Version\", \"buildName\": \"$SOLANA_VERSION\" }" >> $EXECUTOR_FILE
 
-        echo "SOLANA_EXPLORER=\"$SOLANA_EXPLORER\"" >> $ENV_PROP_FILE
```
Comment (apetrovskiy) [inline .github/actions/allure-environment-data/action.yml:52]: solana explorer is also request by Vasily: "this is for everybody not needing to search for it"

**Review by apetrovskiy — CHANGES_REQUESTED** [review]

File: .env.devnet [hunk 2]
```diff
@@ -7,5 +7,5 @@ DISABLE_CONFIRMATION=true
 PRIVATE_KEY=
 FAUCET_QUOTIENT=1
 FAUCET_URL=https://neonswap.live/request_eth_token
-SOLANA_EXPLORER=hhttps://explorer.solana.com/?cluster=custom&customUrl=https%3A%2F%2Fproxy.teststand.neontest.xyz%2Fnode-solana
```
Comment (apetrovskiy) [inline .env.devnet:10]: image

Comment (gigimon) [reply .env.devnet:10]: криво подмерджил )

**Pull request merged**